core/themes/basis/css/skin.css

```diff
@@ -534,7 +534,10 @@ input.form-submit:active {
 }
 
 .button,
-input.form-submit,
+input.form-submit{
```

Comment:
we need to include a space in front of the `{`

Reply:
@thivyaa-web-dev  will make the changes

Reply:
@jenlampton  I made the changes as suggested by you.